apps/kyb-app/jest.config.ts

```diff
@@ -1,5 +1,9 @@
 const config = {
   setupFilesAfterEnv: ['<rootDir>/tests/setup.js'],
+  moduleNameMapper: {
+    '^@app(.*)$': '<rootDir>/src/$1',
+  },
```

Comment:
`kyb-app` should not have Jest as it uses Vitest. Second, we no longer use `@app` and use `@/` instead.